py3dtilers/Common/lod_tree.py

```diff
@@ -38,95 +39,46 @@ def set_centroid(self, centroid):
         self.centroid = centroid
 
 
-def create_lod_tree(objects_to_tile=list(), group=True):
-
-    nodes = list()
-
-    if not group:
-        for object_to_tile in objects_to_tile:
-            nodes.append(LodNode([object_to_tile]))
-    else:
-        nodes.append(LodNode(objects_to_tile))
-
-    tree = LodTree(nodes)
+def create_lod_tree(objects_to_tile, also_create_lod1=False, also_create_loa=False, loa_path=None, with_texture=False):
```

Comment:
Again, I seem to fail to understand the general logic of such function floating around (as opposed to being a member of some class)